dotnet/src/Connectors/Connectors.HuggingFace/HuggingFaceKernelBuilderExtensions.cs

```diff
@@ -43,6 +44,33 @@ public static class HuggingFaceKernelBuilderExtensions
         return builder;
     }
 
+    /// <summary>
+    /// Adds an Hugging Face chat completion service with the specified configuration.
+    /// </summary>
+    /// <param name="builder">The <see cref="IKernelBuilder"/> instance to augment.</param>
+    /// <param name="model">The name of the Hugging Face model.</param>
+    /// <param name="endpoint">The endpoint URL for the chat completion service.</param>
+    /// <param name="apiKey">The API key required for accessing the Hugging Face service.</param>
+    /// <param name="serviceId">A local identifier for the given AI service.</param>
+    /// <param name="httpClient">The HttpClient to use with this service.</param>
+    /// <returns>The same instance as <paramref name="builder"/>.</returns>
+    public static IKernelBuilder AddHuggingFaceChatCompletion(
+        this IKernelBuilder builder,
+        string model,
+        Uri? endpoint = null,
+        string? apiKey = null,
+        string? serviceId = null,
+        HttpClient? httpClient = null)
+    {
+        Verify.NotNull(builder);
+        Verify.NotNull(model);
+
+        builder.Services.AddKeyedSingleton<IChatCompletionService>(serviceId, (serviceProvider, _) =>
+            new HuggingFaceChatCompletionService(model, endpoint, apiKey, HttpClientProvider.GetHttpClient(httpClient, serviceProvider)));
```

Comment:
`ILoggerFactory` is not passed here. It looks like it's not passed in other methods in this file as well. We should probably add it in all extension methods where we initialize new Hugging Face service.
```suggestion
            new HuggingFaceImageToTextService(model, endpoint, apiKey, HttpClientProvider.GetHttpClient(httpClient, serviceProvider), serviceProvider.GetService<ILoggerFactory>()));
```